Index: dnstap_receiver-0.0.1/setup.py
===================================================================
--- dnstap_receiver-0.0.1.orig/setup.py
+++ dnstap_receiver-0.0.1/setup.py
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 
 import setuptools
-from dnstap_receiver import __version__
+#from dnstap_receiver import __version__
 
 with open("README.md", "r") as fh:
     LONG_DESCRIPTION = fh.read()
@@ -10,7 +10,7 @@ KEYWORDS = ('dnstap receiver client json
 
 setuptools.setup(
     name="dnstap_receiver",
-    version=__version__,
+    version="0.0.1",
     author="Denis MACHARD",
     author_email="d.machard@gmail.com",
     description="Python Dnstap to JSON stream receiver",
